RFS/src/main/java/com/rfs/version_universal/RemoteReaderClient_ES_6_8.java

```diff
@@ -0,0 +1,19 @@
+package com.rfs.version_universal;
+
+import java.util.Map;
+
+import com.rfs.common.http.ConnectionContext;
+
+public class RemoteReaderClient_ES_6_8 extends RemoteReaderClient {
```

Comment:
Should this class live with the other ES 6.8 versioned types?  Feels odd to have it live in the "universal" package.